vllm/engine/arg_utils.py

```diff
@@ -1521,15 +1515,25 @@ def _set_default_args_v0(self, model_config: ModelConfig) -> None:
         if self.max_num_seqs is None:
             self.max_num_seqs = 256
 
-    def _set_default_args_v1(self, usage_context: UsageContext) -> None:
+    def _set_default_args_v1(self, usage_context: UsageContext,
+                             model_config: ModelConfig) -> None:
         """Set Default Arguments for V1 Engine."""
 
-        # V1 always uses chunked prefills.
-        self.enable_chunked_prefill = True
+        # V1 always uses chunked prefills and prefix caching
+        # for non-pooling tasks.
+        # For pooling tasks the default is False
```

Comment:
Can we enable prefix caching and chunked prefills by default for last-pooling models?

Reply:
Yes, it's straightforward while only support decoder models. After we add encoder models it will be a bit trickier.